persistence/internal/providertest/aggregateop.go

```diff
@@ -213,5 +215,140 @@ func declareAggregateOperationTests(tc *TestContext) {
 				))
 			})
 		})
+
+		ginkgo.Describe("type persistence.SaveAggregateSnapshot", func() {
+			ginkgo.When("the instance does not exist", func() {
```

Comment:
Just a note for improvements to terminology throughout these tests - "instance" refers to the concept of the "aggregate instance" as defined  in [dogmatiq/dogma](https://github.com/dogmatiq/dogma#aggregate), not just the generic sense of an "instance of an object", so it's more precise use language like:

```suggestion
			ginkgo.When("there is no snapshot of the instance", func() {
```